platform/engineering/design-docs/2021-01-13-web-component-library.md

```diff
@@ -0,0 +1,139 @@
+# Web Component library
+
+**Author(s):** Brooks Johnson  
+**Last Updated:** January 13, 2021  
+**Status:** **Draft** | In Review | Approved  
+**Approvers:** _Chris Valarida_ \[ \], _Michael Fleet_ \[ \]
+
+
+## Overview
+
+### Objective
+
+The goal is to establish a component library based on Web Components, which are a browser standard. This will supercede the current `formation-react` library and some of the liquid HTML templates currently in `vets-website`. A non-goal is creating an entirely new visual experience as part of the component library.
```

Comment:
1. Fun fact (which I just learned): `supercede` is apparently a "disputed variant spelling" of supersede.
2. Additionally, we can probably replace `formation-react` with `component-library` now.
3. Is the goal to completely replace the existing component library with web components?

Reply:
3. I would like that to be the goal.